### PR TITLE
Add config option to ignore cached queries from being counted

### PR DESCRIPTION
**Pull request by rafaelsales**
Let me know if the idea is welcome so I can write tests

Closes https://github.com/brigade/db-query-matchers/issues/26

### CODE REVIEW

**Comment by rafaelsales** [comment]
@lencioni @trotzig can you guys let me know if I can proceed adding specs?

**Comment by lencioni** [comment]
It sounds good to me but I am also not really involved in this project anymore so I'll let @trotzig chime in.

Would you be interested in helping to maintain this project going forward?

**Comment by trotzig** [comment]
Sorry for the delay, I’ve been out of the loop for a few days. Your change looks good to me.

**Comment by rafaelsales** [comment]
@lencioni sure, I don't have a lot of OSS in my plate, so I can definitely help.
@trotzig Added specs and changelog entry!

File: spec/db_query_matchers/make_database_queries_spec.rb [hunk 1]
```diff
@@ -256,6 +256,39 @@
...
+    context 'with ActiveRecord cache' do
+      let(:pattern) { /SELECT.*FROM.*cats/ }
+      subject do
+        ActiveRecord::Base.connection.cache do
+          2.times { Cat.first }
+        end
+      end
+
+      context 'when a `ignore_cached` option is true' do
+        before do
```
Comment (kbrock) [inline spec/db_query_matchers/make_database_queries_spec.rb:268]: You probably want an `after {}` to reset `ignored_cached` back to `false`.
Don't want a change in this value to affect other specs.

NOTE: I do not have merge rights - so don't feel you need to make these changes

**Comment by rafaelsales** [comment]
Given up

**Pull request closed without merging**